virttest/utils_test/qemu/__init__.py

```diff
@@ -204,10 +204,21 @@ def _check_driver_stat():
 
     for drv in driver.split():
         error_context.context("Check %s driver state." % drv, LOG.info)
-        driver_check_cmd = (
-            r'wmic sysdriver where PathName="C:\\Windows\\System32'
-            r'\\drivers\\%s.sys" get State /value'
-        ) % drv
+        driver_svc_map = {
+            "viorng": "VirtRng",
+            "vioser": "VirtioSerial",
+            "viofs": "VirtioFsDrv",
+            "vioinput": "VirtioInput"
+        }
+        if drv in list(driver_svc_map.keys()):
+            driver_svc = driver_svc_map[drv]
+        else:
+            driver_svc = drv
```

Comment:
```suggestion
        driver_svc = driver_svc_map[drv] if drv in driver_svc_map.keys() else drv
```


In python3, dict_keys object is iterable, it support to using `in` operator. In python2, the `keys()` method will return a `list` object directly.So I believe we do no need to convert the type.Thanks.